thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -674,9 +712,9 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
                  */
                 ForEachItemIn(t, targets)
                 {
-                    CTarget *target = targets.item(t);
-                    if (target->getSenderFinished())
+                    if (getSenderFinished(t))
```

Comment:
Is this the correct test for the all case?  the parameter is a node number, but this will only be called with 0.  I suspect it works.

Reply:
No looks wrong.
It probably shouldn't be entering checkSendersFinished at all in the All case. Because it can't clear the buckets until all the node targets are done.
I will change.